Makefile.work

```diff
@@ -38,6 +38,13 @@
 #  * SONIC_DPKG_CACHE_METHOD: Specifying method of obtaining the Debian packages from cache: none or cache
 #  * SONIC_DPKG_CACHE_SOURCE: Debian package cache location when cache enabled for debian packages
 #  * BUILD_LOG_TIMESTAMP: Set timestamp in the build log (simple/none)
+#  * SONIC_VERSION_CONTROL_COMPONENTS : In the process of SONiC build, there are some source code,
```

Comment:
Suggest removing `SONIC_` prefix from user-configurable variable for brevity.

Reply:
thanks~

Reply:
keep prefix for consistent with existing parameter. this parameter mainly used in configure file.